src/frontend/app/shared/components/create-application/create-application-step1/create-application-step1.component.html

```diff
@@ -14,19 +14,20 @@
         <span>{{ org.name }}</span>
       </mat-option>
     </mat-select>
-    <mat-error *ngIf="!(cfOrgSpaceService.space.list$ | async)?.length">
+    <mat-error *ngIf="!(hasSpaces$ | async)">
       There are no organizations in this Cloud Foundry
     </mat-error>
   </mat-form-field>
   <mat-form-field>
     <mat-select #spaceSelect="ngModel" [disabled]="!cfOrgSpaceService.org.select.getValue() || (cfOrgSpaceService.space.loading$ | async) || isRedeploy" [ngModel]="cfOrgSpaceService.space.select.getValue()" (selectionChange)="cfOrgSpaceService.space.select.next($event.value)"
       required placeholder="Space" name="space">
-      <mat-option *ngFor="let space of cfOrgSpaceService.space.list$ | async" [value]="space.guid">
+      <mat-option *ngFor="let space of spaces$ | async" [value]="space.guid">
         <span>{{ space.name }}</span>
       </mat-option>
     </mat-select>
-    <mat-error *ngIf="!(cfOrgSpaceService.space.list$ | async)?.length">
+    <mat-error *ngIf="!(hasSpaces$ | async)">
       There are no spaces in this organization
     </mat-error>
   </mat-form-field>
+  <p>Not seeing your spaces? Please make sure you have the relevant space permissions.</p>
```

Comment:
Feels odd to show this all the time when for the majority of the time it's not needed. Can it be pushed, in an condensed form, into the `no spaces` error message

Reply:
It is a broader message than "no spaces". There may well be spaces but not the ones you were expecting.